registration/templates/registration/password_reset_complete.html

```diff
@@ -6,7 +6,7 @@
 {% block content %}
 <p>
     {% trans "Your password has been reset!" %}
-    {% trans "You may now" %}<a href="{{ login_url }}">{% trans "log in" %}</a>.
+    {% trans "You may now" %} <a href="{{ login_url }}">{% trans "log in" %}</a>.
```

Comment:
I'm a collaborator. :smile: @macropin felt comfortable with giving me collaborator privileges after contributing a couple pull requests. I use django-registration-redux in production and decided to take a more active approach after being underwhelmed with user registration options in Django.